packages/web-app-draw-io/src/App.vue

```diff
@@ -47,6 +47,11 @@ export default defineComponent({
       } = this.applicationConfig
       return { url, theme, autosave: autosave ? 1 : 0 }
     },
+    urlHost() {
+      const url = new URL(this.config.url)
+      const urlHost = `${url.protocol}//${url.hostname}`
```

Comment:
Does protocol contain a `:`? 
Looks like it's missing here

Reply:
Yep, see https://developer.mozilla.org/en-US/docs/Web/API/URL/protocol

Reply:
Surprising, alright then!